lawnchair/src/ch/deletescape/lawnchair/smartspace/weathercom/Constants.kt

```diff
@@ -0,0 +1,82 @@
+                                          17 to WeatherIconProvider.CONDITION_SNOW,
+                                          18 to WeatherIconProvider.CONDITION_SNOW,
+                                          19 to WeatherIconProvider.CONDITION_MIST,
+                                          21 to WeatherIconProvider.CONDITION_MIST,
+                                          22 to WeatherIconProvider.CONDITION_MIST,
+                                          23 to WeatherIconProvider.CONDITION_CLOUDS,
+                                          24 to WeatherIconProvider.CONDITION_CLOUDS,
+                                          25 to WeatherIconProvider.CONDITION_CLOUDS,
+                                          23 to WeatherIconProvider.CONDITION_CLOUDS,
+                                          24 to WeatherIconProvider.CONDITION_CLOUDS,
+                                          25 to WeatherIconProvider.CONDITION_CLOUDS,
+                                          26 to WeatherIconProvider.CONDITION_MOST_CLOUDS,
+                                          27 to WeatherIconProvider.CONDITION_MOST_CLOUDS,
+                                          28 to WeatherIconProvider.CONDITION_CLOUDS,
+                                          29 to WeatherIconProvider.CONDITION_CLOUDS,
+                                          30 to WeatherIconProvider.CONDITION_CLOUDS,
+                                          31 to WeatherIconProvider.CONDITION_CLEAR,
+                                          32 to WeatherIconProvider.CONDITION_CLEAR,
+                                          33 to WeatherIconProvider.CONDITION_MOST_CLOUDS,
+                                          34 to WeatherIconProvider.CONDITION_MOST_CLOUDS,
+                                          35 to WeatherIconProvider.CONDITION_RAIN,
+                                          36 to WeatherIconProvider.CONDITION_CLEAR,
+                                          37 to WeatherIconProvider.CONDITION_STORM,
+                                          38 to WeatherIconProvider.CONDITION_STORM,
+                                          39 to WeatherIconProvider.CONDITION_SHOWERS,
+                                          40 to WeatherIconProvider.CONDITION_RAIN,
+                                          41 to WeatherIconProvider.CONDITION_RAIN,
+                                          42 to WeatherIconProvider.CONDITION_SNOW,
+                                          43 to WeatherIconProvider.CONDITION_SNOW,
+                                          44 to WeatherIconProvider.CONDITION_UNKNOWN,
+                                          45 to WeatherIconProvider.CONDITION_SHOWERS,
+                                          46 to WeatherIconProvider.CONDITION_SNOW,
+                                          47 to WeatherIconProvider.CONDITION_STORM)
+        val WEATHER_ICONS_DAY = WEATHER_ICONS.map { it.key to it.value + "d" }
```

Comment:
Just removing "d" and "n" from the icon code at the call site would probably work as well as the API already supplies us with day/night info

Reply:
I’ll take a look at that, although it is due to the fact that the API sometimes doesn’t adhere to API guidelines